src/annotations/autolinks.ts

```diff
@@ -440,24 +448,42 @@ function ensureCachedRegex(
 ): asserts ref is RequireSome<CacheableAutolinkReference, 'messageRegex'>;
 function ensureCachedRegex(ref: CacheableAutolinkReference, outputFormat: 'html' | 'markdown' | 'plaintext') {
 	// Regexes matches the ref prefix followed by a token (e.g. #1234)
+	function escapeRegexCharacters(regex: string, fn: (c: string) => string) {
+		return regexpTree.transform(`/${ regex }/`, {
```

Comment:
I'm wondering if it would be too hard to achieve the same result without having to add a new dependency to vscode-gitlens 🤔

regexp-tree is really cool, nonetheless.

Reply:
I tried very hard and could not find any other way sadly.

Actually it took me a long way to reach this state even.

lol